### PR TITLE
IZE-403 include new config struct

### PR DESCRIPTION
**Pull request by psihachina**
## Changelog:
- include new config struct
- refactoring
- some fixes
- rename apps package to manager

## Test:
[asciicast](https://asciinema.org/a/YrwJ2HAjGGcDHSmR4ssFQD1Ee)

```
 ize up squibby   

» Deploying squibby app...

⠙ squibby: building app container...
✓ squibby: building app container...
 │ Step 12/12 : EXPOSE $PORT
 │  ---> Using cache
✓ squibby: building app container...
 │ Step 12/12 : EXPOSE $PORT
 │  ---> Using cache
 │  ---> 7adaae8c6cda
 │ [Warning] One or more build-args [APP_NAME] were not consumed
 │ Successfully built 7adaae8c6cda
 │ Successfully tagged testnut-squibby:latest
 │ Successfully tagged 322403564058.dkr.ecr.us-east-1.amazonaws.com/testnut-squibby
 │ :1ac14c2
 │ Successfully tagged 322403564058.dkr.ecr.us-east-1.amazonaws.com/testnut-squibby
 │ :testnut-latest
✓ squibby: push app image...
 │ b9dd76c189d6: Layer already exists
 │ 7d3669461155: Layer already exists
 │ 8384c7098c01: Layer already exists
 │ 4531d365dde0: Layer already exists
 │ 9e567d1bb30a: Layer already exists
 │ 56bb29f4ee33: Layer already exists
 │ 09c126bb3acd: Layer already exists
 │ 24302eb7d908: Layer already exists
 │ testnut-latest: digest: sha256:8de568d2a1bc153f7cc334b262978e478c597e04bee7932a5
 │ f897240a4ed8ff5 size: 2836
✓ squibby: deploying app container...
 │ Changed image of container "squibby" to : "322403564058.dkr.ecr.us-east-1.amazon
 │ aws.com/testnut-squibby:testnut-latest" (was: "322403564058.dkr.ecr.default.amaz
 │ onaws.com/testnut-squibby:testnut")
 │ Creating new task definition revision
 │ Successfully created revision: testnut-squibby:377
 │ Updating service
 │ Successfully changed task definition to: testnut-squibby:377
 │ Deploying new task definition
 │ Deregister task definition revision
 │ Successfully deregistered revision: testnut-squibby:376
✓ squibby: deployment completed!
Deploy app squibby completed

```

```
ize down infra 

» Running terraform destroy...
✓ Running terraform v1.1.7...
 │ aws_security_group.default_permissive: Destruction complete after 21s
 │ module.vpc.aws_internet_gateway.this[0]: Still destroying... [id=igw-058babe2eb7
 │ 6b7e75, 1m10s elapsed]
 │ module.vpc.aws_internet_gateway.this[0]: Destruction complete after 1m15s
 │ module.vpc.aws_vpc.this[0]: Destroying... [id=vpc-0517e8922e57c56bc]
 │ module.vpc.aws_vpc.this[0]: Destruction complete after 2s
 │ Releasing state lock. This may take a few moments...
 │ 
 │ Destroy complete! Resources: 58 destroyed.
 │ 
Terraform destroy completed!

```

https://github.com/hazelops/ize/runs/7305967401?check_suite_focus=true

### CODE REVIEW

File: internal/apps/dependencies.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func InDependencyOrder(ctx context.Context, apps map[string]*interface{}, fn fun
 	return visit(ctx, apps, upDirectionTraversalConfig, fn, AppStopped)
 }
 
-// InReverseDependencyOrder applies the function to the apps of the project in reverse order of dependencies
+// InReversDependencyOrder InReverseDependencyOrder applies the function to the apps of the project in reverse order of dependencies
```
Comment (kobrikx) [inline internal/apps/dependencies.go:58]: Why do you remove `e` from Reverse?

File: internal/commands/tunnel/tunnel_sshkey.go [hunk 2]
```diff
@@ -98,8 +90,7 @@ func (o *TunnelSSHKeyOptions) Run() error {
 		return fmt.Errorf("can't send ssh key: %w", err)
 	}
 
-	s.Done()
-	ui.Output("SSH user's public key has been sent!\n", terminal.WithSuccessStyle())
+	pterm.Success.Printfln("SSH user's public key has been sent!\n", terminal.WithSuccessStyle())
```
Comment (kobrikx) [inline internal/commands/tunnel/tunnel_sshkey.go:93]:
```suggestion
	pterm.Success.Printfln("User SSH public key has been sent!\n", terminal.WithSuccessStyle())
```

**Review by kobrikx — APPROVED** [review]
Test looks good. I have a couple of comments.

File: examples/ecs-apps-monorepo/.ize/env/testnut/ize.toml [hunk 3]
```diff
@@ -4,8 +4,8 @@ terraform_version = "1.1.7"
 
 [app.squibby]
 type = "ecs"
-path = "./projects/squibby"
+path = "./apps/squibby"
```
[the remark this answers is not in the document]
Comment (psihachina) [reply examples/ecs-apps-monorepo/.ize/env/testnut/ize.toml:7]: Yes, has been renamed. This is fix.

**Review by AutomationD — APPROVED** [review]

**Pull request merged**